tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -156,60 +158,71 @@ def test_quant_model_reload(format, dtype, tmp_path):
     dataset = "open_platypus"
     concatenate_data = False
     num_calibration_samples = 64
-    output_dir = tmp_path / "oneshot_out"
     splits = {"calibration": "train[:10%]"}
+    empty_model = AutoModelForCausalLM.from_pretrained(model_path, torch_dtype=dtype)
```

Comment:
Eventually, It would be nice if decompression worked with an actual empty model with tensors on a `meta` device, but this is good for the time being!

Reply:
I agree, it'd be nicer if `model_compressor.decompress(path)` returned the decompressed AutoModel instance